### PR TITLE
[14.0][MIG] sale_order_invoicing_queued: Migration to 14.0

### PR DESCRIPTION
**Pull request by iwkse**
@pedrobaeza That's the porting using the XML instead of the decorator, but it doesn't seem to work. I hope the PR so that we can proceed with the module,
Thank you!

### CODE REVIEW

**Comment by iwkse** [comment]
@pedrobaeza please when it's possible give me a feedback
Thank you

File: sale_order_invoicing_queued/data/job_queue.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<odoo noupdate="1">
+  <record id="job_function_sale_order_invoicing_action_done" model="queue.job.function">
+    <field name="model_id" ref="sale.model_sale_order" />
+    <field name="method">create_invoices_job</field>
+    <field name="related_action" eval='{"func_name": "custom_related_action"}' />
```
Comment (pedrobaeza) [inline sale_order_invoicing_queued/data/job_queue.xml]: Is this one correct? (I don't know the syntax)

**Review by pedrobaeza — APPROVED** [review]

File: sale_order_invoicing_queued/data/job_queue.xml (hunk 1 above)
Comment (iwkse) [reply sale_order_invoicing_queued/data/job_queue.xml]: That's from the example provided in the README file

Comment (pedrobaeza) [reply sale_order_invoicing_queued/data/job_queue.xml]: But I think you should provide a valid action name. If not, I would say to remove it and it should take the default one.

Comment (pedrobaeza) [reply sale_order_invoicing_queued/data/job_queue.xml]: I was talking about this line, not the whole file.

**Comment by pedrobaeza** [comment]
Travis is red

**Comment by iwkse** [comment]
> Travis is red
Yes, when removing the action it requires the channel_id configuration I think. I'm going to fix that asap

**Comment by iwkse** [comment]
> Travis is red

fixed it

**Review by pedrobaeza — APPROVED** [review]

**Review by HaraldPanten — CHANGES_REQUESTED** [review]
Couldn't make it work in runbot. Steps:

1- I Created a sales order with an invoiceable line.
2- I went to the tree view to create the invoice and I selected to enqueue it in the wizard.
3- The job seems to be enqueued, but it never gets finished and the invoice is not created.

Am I missing anything?

THX

**Comment by pedrobaeza** [comment]
Runbot is not configured for running jobs, so that's the problem. It's not something of this module.

**Review by HaraldPanten — APPROVED** [review]
LGTM 👍

**Pull request merged**